### PR TITLE
Adding separate driver for United States customary units

### PR DESCRIPTION
**Pull request by martinichka**
- our drivers are by default using SI units, this driver is a duplicate of the VB driver, but than using USC units

### CODE REVIEW

**Comment by martinichka** [comment]
@norbertherbert @mostafaibr this is the extra driver for the VB with the USC units as discussed in our last meeting.

File: vendors/twtg/models/neon-vb-usc/model.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+# <vendorId>:<protocolName>:<protocolVersion> Needed for linking the model with a specific driver -> must be the same one used in driver.yaml in the corresponding model (You might have several ones)
+protocolId: twtg:neon-vb-usc:3
+# DataSheet URL (optional) 
+specificationURL: https://github.com/TWTG-R-D-B-V/neon-product-documentation/blob/main/VB/6016_P20-002_Data-Sheet-NEON-Vibration-Sensor_C.pdf
+# User Guide URL (optional) 
+userGuideURL: https://github.com/TWTG-R-D-B-V/neon-product-documentation/tree/main/VB/Installation%20%26%20Use
+# Available sensors following Actility ontology: https://github.com/actility/thingpark-iot-flow-js-driver/blob/master/UNITS.md 
+sensors: 
+    - temperature:Far
+    - velocity:in/s
```
Comment (martinichka) [inline vendors/twtg/models/neon-vb-usc/model.yaml:41]: `in` and `ft` were not listed, so I don't know if this is allowed. If not, I can just remove them.

Comment (mostafaibr) [reply vendors/twtg/models/neon-vb-usc/model.yaml:41]: Hello @martinichka ,

At the moment, we can keep sensors compatibles with the existing ontology, so please remove the ones which are not listed. In the future, we can enhance this part by adding first these units to the ontology.

Comment (martinichka) [reply vendors/twtg/models/neon-vb-usc/model.yaml]: I removed the non-existing unit

File: vendors/twtg/models/neon-vb-usc/model.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+# Typical mobility profile of the device. Possible values are 'near_static' (also valid for static devices), 'walking_speed', 'vehicular_speed'  
+# or 'random' (not known, changes over time). 
+motionIndicator: NEAR_STATIC
+ 
+# Is your device certified by the LoRa Alliance? Possible values: true, false. 
+LoRaWANCertified: true
+# Always mandatory: `<vendorId>:<modelName>:<modelVersion>` (You should have only one per model)
+modelId: twtg:neon-vb-usc:1
+# <vendorId>:<protocolName>:<protocolVersion> Needed for linking the model with a specific driver -> must be the same one used in driver.yaml in the corresponding model (You might have several ones)
+protocolId: twtg:neon-vb-usc:3
```
[the remark this answers is not in the document]
Comment (martinichka) [reply vendors/twtg/models/neon-vb-usc/model.yaml]: I removed the USC model and refered the USC driver from the existing model

**Comment by mostafaibr** [comment]
Thank you @martinichka  for the modifications. i reviewed the drivers and models, and will merge your PR. Your drivers would be available soon on Thingpark.

**Pull request merged**